Projects/App/Sources/Journeys/LoginNavigation.swift

```diff
@@ -86,24 +84,16 @@ public struct NotLoggedInView: View {
     }
 
     @ViewBuilder
-    var marketAndLanguage: some View {
+    var language: some View {
```

Comment:
we could actually name this one languageView for readability just to be extra clear that this is a view!

Reply:
Makes sense! I'll fix it!